lib/virt_autotest/kernel.pm

```diff
@@ -37,7 +37,7 @@ sub check_virt_kernel {
     my $suffix       = $args{suffix}   // '';
     my $log_file     = $args{log_file} // '/tmp/virt_kernel.txt';
     my $go_to_target = $target eq 'localhost' ? '' : "ssh root\@$target";
-    my ($sles_running_version, $sles_running_sp) = get_sles_release($go_to_target);
+    my ($sles_running_version, $sles_running_sp) = get_os_release($go_to_target);
```

Comment:
This is something I see multiple times in the code.
The subroutine now returns different variables.

Reply:
What do you mean different variables? `$sles_running_version, $sles_running_sp` are exactly the same. the subroutine returns an additional one which identicates the `id` aka distro name. the scripts that use the subroutine are not affected and it is up to maintainers of the module to modify them to leverage the extra variable.

Reply:
But the additional variable is returned as the first one.
In this case, the `sles_running_version` will be `sles` and `sles_running_sp` will be `15-SP2`.

Reply:
```
$ cat /tmp/x
sub asd {
  my $a = 'a';
  my $b = 'b';
  my $c = 'c';
  return $a, $b, $c;
}

my ($b, $c) = asd();
print($b.$c."\n");

$ perl /tmp/x
ab
```

Reply:
the output from the example looks the expected. where is the *'c'* as a first variable? if you use two variables, as you do the result is 'a' and 'b'

Reply:
in brief `my ($running_sles_rel, $running_sles_sp) = get_os_release` behaves still the same.

Reply:
http://pdostal-server.suse.cz/tests/10264#